x-pack/filebeat/module/okta/system/test/okta-system-test.json.log-expected.json

```diff
@@ -259,5 +266,215 @@
+        "event.outcome": "success",
+        "fileset.name": "system",
+        "input.type": "log",
+        "log.offset": 7707,
+        "okta.actor.alternate_id": "xxxxxx@elastic.co",
+        "okta.actor.display_name": "xxxxxx",
+        "okta.actor.id": "00u1abvz4pYqdM8ms4x6",
+        "okta.actor.type": "User",
+        "okta.authentication_context.authentication_step": 0,
+        "okta.authentication_context.external_session_id": "102bZDNFfWaQSyEZQuDgWt-uQ",
+        "okta.client.device": "Computer",
+        "okta.client.ip": "108.255.197.247",
+        "okta.client.user_agent.browser": "FIREFOX",
+        "okta.client.user_agent.os": "Mac OS X",
+        "okta.client.user_agent.raw_user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:72.0) Gecko/20100101 Firefox/72.0",
+        "okta.client.zone": "null",
+        "okta.debug_context.debug_data.request_id": "<random_id_string>",
+        "okta.debug_context.debug_data.request_uri": "<uri_endpoint>",
+        "okta.debug_context.debug_data.suspicious_activity.browser": "browser",
+        "okta.debug_context.debug_data.suspicious_activity.event_city": "New York City",
+        "okta.debug_context.debug_data.suspicious_activity.event_country": "United States",
+        "okta.debug_context.debug_data.suspicious_activity.event_id": "1234567",
```

Comment:
It personally doesn't matter to me. I just figured it was easier to read the fields since they were all `suspiciousActivityXXXX`. @BenB196, any thoughts from u since ur the original reporter of the issue?

Reply:
I don't think it would be an issue. My only "concern" would be if Okta were to ever add anything to the ```debug_context``` level. With the removal of ```debug_data``` you could potentially have field overlap/collision. Not sure the likelihood of this though.

Reply:
After some more discussion, its better to leave it consistent with the rest of the debug data, so we will keep it like this for now